draft-ietf-lamps-csr-attestation.md

```diff
@@ -196,7 +194,7 @@ or physically separated from the core CA functionality.
  .------------.               .----|----------.
  |            +-------------->|----'          | Compare Attestation
  |  Attester  |   Evidence    | Relying       | Result against
- |            |   in CSR      | Party (RA/CA) | policy
+ |   (HSM)    |   in CSR      | Party (RA/CA) | policy
```

Comment:
```suggestion
 |  (with HSM)    |   in CSR      | Party (RA/CA) | policy
```

Reply:
The HSM is not the Attstester, just a part of one of the Attester's Attesting Environments